### PR TITLE
init

### PR DESCRIPTION
**Pull request by ttossavainen**
https://github.com/FrendsPlatform/Frends.AmazonSQS/issues/1

### CODE REVIEW

File: Frends.AmazonSQS.Send/Frends.AmazonSQS.Send/Definitions/Result.cs [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    public bool Success { get; private set; }
+
+    /// <summary>
+    /// MessageId.
+    /// </summary>
+    /// <example>aae72806-1469-4663-b3d3-5228d163baa6</example>
+    public string MessageId { get; private set; }
+
+    /// <summary>
+    /// HTTP status code of the operation.
```
Comment (RikuVirtanen) [inline Frends.AmazonSQS.Send/Frends.AmazonSQS.Send/Definitions/Result.cs]: Is 'OK' a HTTP status code? You might need to change the description.

File: Frends.AmazonSQS.Send/README.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# Frends.AmazonSQS.Send
+Frends Task for executing AmazonSQS queries.
```
Comment (RikuVirtanen) [inline Frends.AmazonSQS.Send/README.md]: This should be the same as in the Task's main methods description.

File: Frends.AmazonSQS.Send/Frends.AmazonSQS.Send/Frends.AmazonSQS.Send.csproj [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+	<TargetFrameworks>net6.0</TargetFrameworks>
+	<Version>1.0.0</Version>
+	<Authors>Frends</Authors>
+	<Copyright>Frends</Copyright>
+	<Company>Frends</Company>
+	<Product>Frends</Product>
+	<PackageTags>Frends</PackageTags>
+	<PackageLicenseExpression>MIT</PackageLicenseExpression>
+	<GenerateDocumentationFile>true</GenerateDocumentationFile>
+	<Description>Frends Task for executing AmazonSQS queries.</Description>
```
Comment (RikuVirtanen) [inline Frends.AmazonSQS.Send/Frends.AmazonSQS.Send/Frends.AmazonSQS.Send.csproj]: This should be the same as in the Task's main methods description.

File: Frends.AmazonSQS.Send/README.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+[![License: MIT](https://img.shields.io/badge/License-MIT-green.svg)](https://opensource.org/licenses/MIT) 
+[![Build](https://github.com/FrendsPlatform/Frends.AmazonSQS/actions/workflows/ExecuteQuery_build_and_test_on_main.yml/badge.svg)](https://github.com/FrendsPlatform/Frends.AmazonSQS/actions)
```
Comment (RikuVirtanen) [inline Frends.AmazonSQS.Send/README.md]: yml reference is incorrect.

File: Frends.AmazonSQS.Send/Frends.AmazonSQS.Send.sln [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Frends.AmazonSQS.Send", "Frends.AmazonSQS.Send\Frends.AmazonSQS.Send.csproj", "{35C305C0-8108-4A98-BB1D-AFE5C926239E}"
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution Items", "{78F7F22E-6E20-4BCE-8362-0C558568B729}"
+	ProjectSection(SolutionItems) = preProject
+		CHANGELOG.md = CHANGELOG.md
+		README.md = README.md
+		..\.github\workflows\ExecuteQuery_build_and_test_on_main.yml = ..\.github\workflows\ExecuteQuery_build_and_test_on_main.yml
+		..\.github\workflows\ExecuteQuery_release.yml = ..\.github\workflows\ExecuteQuery_release.yml
+		..\.github\workflows\ExecuteQuery_build_and_test_on_push.yml = ..\.github\workflows\ExecuteQuery_build_and_test_on_push.yml
```
Comment (RikuVirtanen) [inline Frends.AmazonSQS.Send/Frends.AmazonSQS.Send.sln]: workflow references are incorrect.

**Review by RikuVirtanen — CHANGES_REQUESTED** [review]

File: Frends.AmazonSQS.Send/Frends.AmazonSQS.Send/Definitions/Result.cs (hunk 1 above)
Comment (ttossavainen) [reply Frends.AmazonSQS.Send/Frends.AmazonSQS.Send/Definitions/Result.cs]: It is actually. There's no numeric code to get from the response. Removed the "code" part of the description.

**Review by RikuVirtanen — APPROVED** [review]

**Pull request merged**